src/algebra/module/equiv.lean

```diff
@@ -225,7 +222,13 @@ include σ'
 @[simp] lemma inv_fun_eq_symm : e.inv_fun = e.symm := rfl
 omit σ'
 
-@[simp] lemma coe_to_equiv_symm : ⇑e.to_equiv.symm = e.symm := rfl
+@[simp, norm_cast] lemma coe_to_equiv_symm : (e.symm : M₂ ≃ M) = (e : M ≃ M₂).symm := rfl
+
+@[simp, norm_cast]
+lemma coe_to_add_equiv_symm : (e.symm : M₂ ≃+ M) = (e : M ≃+ M₂).symm := rfl
```

Comment:
Why do you reverse the simp direction of `coe_to_equiv_symm`? With the original direction, `⇑(e : M ≃ M₂).symm` could be simplified to `⇑e.symm` by [coe_fn_coe_base](https://leanprover-community.github.io/mathlib_docs/logic/basic.html#coe_fn_coe_base) so `coe_to_equiv_symm_apply` below become redundant.

Reply:
I know all this, but this is the direction that was decided upon on Zulip: https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/simp.20normal.20form.20for.20.60e.2Eto_equiv.2Esymm.60

It also makes sense in that it uses the `symm` from a weaker equiv.

As all/most existing lemmas move `symm` to the outside, I think making the simp set confluent is a more viable fix than trying to turn them all around.

Reply:
Thanks, I missed that thread. But why do you say it's decided? Seems your vote and @eric-wieser 's are on option (1) (which agrees with the current order of `coe_to_equiv_symm`), and you can count me as well (but I've not thought deeply about the issue).